R/cluster.R

```diff
@@ -234,15 +234,15 @@ makeCluster <-
     }
 
     if (!is.null(poolConfig[["pool"]])) {
-      Validators$isValidDeprecatedClusterConfig(clusterSetting)
+      `validation`$isValidDeprecatedClusterConfig(clusterSetting)
```

Comment:
Why are you wrapping validation in ``?

Reply:
Deciding if this is the right approach to differentiate that it's a class and not just a regular list of lists